app/models/tax.rb

```diff
@@ -0,0 +1,5 @@
+module Tax
```

Comment:
don't think we need this, just specify the table_name in the `Tax::Vehicle` class

Reply:
As we use module there automatic create by scaffold. Also it is good if we have other model in that module. So I think we should keep it without specific in model